File: src/main/webapp/osca/common/domain/Referentiel/Rubriques/SousRubriques/UX/FeuilleDeRouteUX.ts
import { CriteresUX } from './CriteresUX';
import { PratiquesUX } from './PratiquesUX';

export interface FeuilleDeRouteUX {
    titre: string;
    keyFeuilleDeRouteUX: string;
    criteresUXList: CriteresUX[];
    pratiquesUXList: PratiquesUX[];
}

File: src/main/java/com/soprasteria/osca/domain/referentiel/rubriques/sousrubriques/ux/FeuilleDeRouteUX.java
package com.soprasteria.osca.domain.referentiel.rubriques.sousrubriques.ux;

import lombok.Getter;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
public class FeuilleDeRouteUX {

    private String titre;
    private String keyFeuilleDeRouteUX;
    private List<CriteresUX> criteresUXList;
    private List<PratiquesUX> pratiquesUXList;
}
